toolchain/lexer/token_kind.cpp

```diff
@@ -107,6 +107,12 @@ auto TokenKind::fixed_spelling() const -> llvm::StringRef {
 auto TokenKind::expected_parse_tree_size() const -> int {
   static constexpr int8_t Table[] = {
 #define CARBON_TOKEN(Name) 1,
+// CARBON_TOKEN_WITH_VIRTUAL_NODE wraps CARBON_TOKEN, so needs to be written
+// as-is.
+// TODO: Consider changing the macro setup to avoid the lint issue, e.g. so that
+// CARBON_KEYWORD_TOKEN would be called by a macro that could be replaced,
+// instead of wrapped.
+// NOLINTNEXTLINE(bugprone-macro-parentheses)
 #define CARBON_TOKEN_WITH_VIRTUAL_NODE(size) 1 + size
```

Comment:
Yeah, I think this is the wrong kind of clever. Perhaps we could do this for now:

```suggestion
#define CARBON_TOKEN_WITH_VIRTUAL_NODE(size) 2,
```

The existing code would support multiple virtual nodes through repeated wrapping with `CARBON_TOKEN_WITH_VIRTUAL_NODE`, but I'm not sure that's a feature that's worth preserving :)